src/cart/crt_swim.c

```diff
@@ -94,27 +103,73 @@ static void crt_swim_srv_cb(crt_rpc_t *rpc_req)
 {
 	struct crt_grp_priv	*grp_priv = crt_gdata.cg_grp->gg_srv_pri_grp;
 	struct crt_swim_membs	*csm = &grp_priv->gp_membs_swim;
+	struct swim_context	*ctx = csm->csm_ctx;
 	struct crt_rpc_swim_in	*rpc_swim_input = crt_req_get(rpc_req);
-	int			 rc;
+	struct crt_rpc_swim2_out *rpc_swim_output = crt_reply_get(rpc_req);
+	swim_id_t		 self_id = swim_self_get(ctx);
+	int			 rc = -ENOTCONN;
 
 	D_ASSERT(crt_is_service());
 
-	D_DEBUG(DB_TRACE, "incoming message %lu <= %lu\n",
-		swim_self_get(csm->csm_ctx), rpc_swim_input->src);
+	D_TRACE_DEBUG(DB_TRACE, rpc_req,
```

Comment:
RPC_TRACE?

Reply:
No. For this macro I have to pass "struct crt_rpc_priv" which is private for SWIM. I don't wont to export this knowledge here. Also this macro don't get me any benefits. All I need here is print the address of rpc_rec structure to have a reference in the future. I'd like to track the particular RPC request across all calls of SWIM.